tools/packaging/packaging.psm1

```diff
@@ -34,9 +34,15 @@ function Start-PSPackage {
         [Switch] $SkipReleaseChecks
     )
 
+    $Script:Options = Get-PSOptions
```

Comment:
If we don't change `$Script:Options`, this line should be moved back to its original position.

Reply:
will fix